infrastructure/docker/docker-compose.yml

```diff
@@ -8,7 +8,11 @@ services:
         build: services/router
         volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
-        network_mode: "host"
+        #network_mode: "host"
```

Comment:
This break multi projet, and IMHO multi project is more important than windows suppot.
Instead, I would put an entry in the cookbook.
I don't want to bloat the code with condition to know what is the current OS and map port or not :/

Reply:
I never use multi-project. But as we already have custom code for Windows support I don't see a problem having both:
- host network always
- ports on windows

I will submit a commit for that.

Reply:
OK, If you manage to do that simply, let's ship it :+1: 

> host network always
but if it does not work on windows ? :thinking:

Reply:
See the new commit for my proposal.